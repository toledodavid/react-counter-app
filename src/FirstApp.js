import React from 'react';
//import React, { Fragment } from 'react';

const FirstApp = () => {
  return (
    // <Fragment>
    //   <h1>Hello world</h1>
    //   <p>My first App</p>
    // </Fragment>

    // This is also a fragment
    <>
      <h1>Hello world</h1>
      <p>My first App</p>
    </>
  );
}

export default FirstApp;