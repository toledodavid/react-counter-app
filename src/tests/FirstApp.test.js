import React from 'react';
import '@testing-library/jest-dom';
//import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import FirstApp from '../FirstApp';

describe('Tests in < FirstApp /> Component', () => {
  // test("It should show a message - Hello, I'm Toledo", () => {
  //   const greeting = "Hello, I'm Toledo";
  //   const { getByText } = render(<FirstApp greeting={greeting} />);
  //   expect(getByText(greeting)).toBeInTheDocument();
  // });

  test('It should show <FirstApp /> component correctly', () => {
    const greeting = "Hello, I'm Toledo";
    const wrapper = shallow( <FirstApp greeting={greeting} /> );

    expect(wrapper).toMatchSnapshot();
  });
});