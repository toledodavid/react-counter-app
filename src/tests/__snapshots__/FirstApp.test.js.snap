// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in < FirstApp /> Component It should show <FirstApp /> component correctly 1`] = `
<Fragment>
  <h1>
    Hello, I'm Toledo
  </h1>
  <p>
    I'm a subtitle
  </p>
</Fragment>
`;
